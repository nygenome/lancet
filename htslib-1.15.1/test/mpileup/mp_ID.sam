@HD	VN:1.6	SO:coordinate
@SQ	SN:z	LN:13
@CO	
@CO	Copyright (c) 2018 Genome Research Ltd.
@CO	
@CO	Permission is hereby granted, free of charge, to any person obtaining
@CO	a copy of this software and associated documentation files (the
@CO	"Software"), to deal in the Software without restriction, including
@CO	without limitation the rights to use, copy, modify, merge, publish,
@CO	distribute, sublicense, and/or sell copies of the Software, and to
@CO	permit persons to whom the Software is furnished to do so, subject
@CO	to the following conditions:
@CO	
@CO	The above copyright notice and this permission notice shall be included
@CO	in all copies or substantial portions of the Software.
@CO	
@CO	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
@CO	OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
@CO	MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
@CO	IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
@CO	CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
@CO	TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
@CO	SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
@CO	
@CO	Various I combinations
@CO	Starting with I isn't handled due to the pileup ordering.
@CO	The record only appears in the pileup list when the first base
@CO	is placed against the reference, which is too late.
@CO	
@CO	                 1
@CO	    123  45678  90123 Depadded base numbering
@CO	ref TAG  CTTAG  CAGGT
@CO	
@CO	s1   AG  CTTAGTT**GGC 7M 2I 2D 3M
@CO	s2   AG  CTTAGTT**    7M 2I 2D
@CO	s3   AG  CTTAGT***    7M 1I 1P 2D
@CO	s4     AA**TAG  CAGG  2I 2D 7M
@CO	s5     *A**TAG  CAGG  1P 1I 2D 7M
@CO	qual 01AB23456CD78901
@CO	
s1	0	z	2	0	7M2I2D3M	*	0	0	AGCTTAGTTGGC	0123456CD901
s2	0	z	2	0	7M2I2D	*	0	0	AGCTTAGTT	0123456CD
s3	0	z	2	0	7M1I1P2D	*	0	0	AGCTTAGT	0123456D
s4	0	z	4	0	2I2D7M	*	0	0	AATAGCAGG	AB4567890
s5	0	z	4	0	1P1I2D7M	*	0	0	ATAGCAGG	B4567890
