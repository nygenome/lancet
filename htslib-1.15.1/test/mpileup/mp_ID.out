z	2	3	^!A^!A^!A
z	3	3	GGG
z	4	5	CCC^!*^!*
z	5	5	TTT**
z	6	5	TTTTT
z	7	5	AAAAA
z	8	5	G+2(TT)-2()G+2(TT)-2()G+2(T*)-2()GG
z	9	5	***CC
z	10	5	**$*$AA
z	11	3	GGG
z	12	3	GG$G$
z	13	1	C$
